packages/account/src/Components/Routes/helpers.ts

```diff
@@ -0,0 +1,39 @@
+import { match, matchPath, RouteProps } from 'react-router';
+import { routes } from '@deriv/shared';
+import { TRouteConfig } from 'Types';
+
+export const normalizePath = (path: string): string => (/^\//.test(path) ? path : `/${path || ''}`); // Default to '/'
```

Comment:
The default returntype for this function is string, you don't need to add a string type to it

Reply:
Will incorporate this change